lib/meilisearch/index.rb

```diff
@@ -74,17 +74,12 @@ def documents(options = {})
     def add_documents(documents, primary_key = nil)
       documents = [documents] if documents.is_a?(Hash)
       http_post "/indexes/#{@uid}/documents", documents, { primaryKey: primary_key }.compact
+      ask = add_documents(documents, primary_key)
```

Comment:
This will create an infinite loop... I encourage you to test the code like is described in the [CONTRIBUTING.md](https://github.com/meilisearch/meilisearch-ruby/blob/main/CONTRIBUTING.md) file before submitting! 😄